recipes/codelldb/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "codelldb" %}
+{% set version = "1.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vadimcn/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 11bee5666a45629e41dd0a8668ba65d422f0511a3c5fc3a47cdf670ef90818c6
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cargo-bundle-licenses
+    - cmake
+    - lldb
+    - libcxx
+    - nodejs
+    - make
+
+test:
+  commands:
+    - codelldb --help
+
+about:
+  home: https://github.com/vadimcn/{{ name }}
```

Comment:
Please remove all the excess use of jinja. Like this the URL is not clickable. This kind of jinja doesn't help and obscure the recipe.